### PR TITLE
Initiating sonic project

### PR DESCRIPTION
**Pull request by ToSeven**

### CODE REVIEW

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Comment by jonathanmetzman** [comment]
> LGTM

Sorry I meant to leave my review confirming you don't want view restrictions.

File: projects/sonic/project.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+language: go
+primary_contact: "dysghappy@163.com"
+auto_ccs:
+  - "byone.heng@google.com"
+fuzzing_engines:
+  - libfuzzer
+sanitizers:
+  - address
+  - memory
+view_restrictions: none
```
[the remark this answers is not in the document]
Comment (DonggeLiu) [reply projects/sonic/project.yaml:12]: @ToSeven Do we mind this? Thanks :)

[the remark this answers is not in the document]
Comment (ToSeven) [reply projects/sonic/project.yaml:12]: Yeah, I think we should restrict issues.

**Pull request merged**